tests/dhcp_rate_test.py

```diff
@@ -0,0 +1,187 @@
+import os
+from click.testing import CliRunner
+from utilities_common.db import Db
+
+import config.main as config
+import show.main as show
+
+
+
+show_interface_dhcp_rate_limit_output = """\
```

Comment:
Show test is missing? And could you add test for show specified interface?

Reply:
Hi @yaqiangz , we have added the show test case now. Please review it.